serverless/cmd/client/client.go

```diff
@@ -245,29 +246,65 @@ func (l *logClientTool) consistencyProof(ctx context.Context, args []string) err
 	return nil
 }
 
-func (l *logClientTool) inclusionProof(ctx context.Context, args []string) error {
+// For the inclusion subcommand, parse the command-line options and arguments to get the entry's
+// hash and index.
+//
+// When the --inclusion_hash option is not provided, the first argument is taken to be the name
+// of a file that will be hashed to look up the node. The file will be read and the merkle hash
+// will be computed. When the --inclusion_hash option is provided, the first argument will instead
+// be the base64-encoded merkle node hash of the node.
+//
+// The entry's index may optionally be provided as an additional argument. If the index is
+// provided, we'll use that index. The node at that index must match the provided entry's contents
+// or hash. If the index is not provided, we'll do a tree lookup to find the node's index.
+//
+// Returns the entry's hash, the entry's index, and an error.
+func (l *logClientTool) inclusionProofArgs(ctx context.Context, args []string) ([]byte, uint64,  error) {
+	var lh []byte
+	var err error
+	
 	if l := len(args); l < 1 || l > 2 {
-		return fmt.Errorf("usage: inclusion <file> [index-in-log]")
+		return nil, 0, fmt.Errorf("usage: inclusion <file or hash> [index-in-log]")
```

Comment:
/file or hash/file or leaf hash/

Reply:
did you miss this one, or do you feel strongly about keeping as-is?

Reply:
I just missed it. I've updated my branch.